.github/workflows/updatejson.yml

```diff
@@ -0,0 +1,27 @@
+name: Housekeeping
+on: 
+  push:
+  schedule:
+  #  Kick off job at midnight on the 1st and the 15th of every month
+  - cron: "0 0 1,15 * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - id: run_py
+        run:  |
+        cd utils
+        python update_json_files
+        cd ..
+      - id: run_git_commands
+        run:  |
+          git add .
+          git config --global user.name 'Philip Wright'
```

Comment:
I have left my own github name and email in here for the automated commits and pushes. However I can change this to be you @hackerkid  (if you like)

Reply:
I think we might probably want to set it to some random bot email. Example
https://github.com/hackerkid/zulip-archive-action-test-2

Reply:
Cool, I'll get started on this

Reply:
I've added a dummy name and email to the yml. Since its just a dummy user created solely inside the github VM for the purposes of the commit messages (example in the picture below), I haven't bothered sticking it in a secret. 

![image](https://user-images.githubusercontent.com/49524546/99187627-a7708180-274f-11eb-90d3-ece94d554eae.png)